.eslintrc.json

```diff
@@ -46,6 +48,7 @@
     "no-prototype-builtins": "off",
     "no-use-before-define": 2,
     "one-var": [2, "never"],
-    "prettier/prettier": "error"
+    "prettier/prettier": "error",
+    "no-unused-vars": "off"
```

Comment:
Turned this off temporarily as
`/* exported variableName */ ` doesn't work when `parserOptions.sourceType` is "module", so eslint was complaining about unused variables 

https://eslint.org/docs/rules/no-unused-vars#exported